sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -152,6 +155,25 @@ export class AssessmentControllerService {
     return disbursementDetails;
   }
 
+  /**
+   * Calculate the sum of 'valueAmount' for each DisbursementValue within an array of DisbursementSchedules.
```

Comment:
Can avoid technical terms(like array) and object names DisbursementSchedules in the comment.